<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FSMainWindow</class>
 <widget class="QMainWindow" name="FSMainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>565</width>
    <height>284</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="gridLayoutWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>551</width>
      <height>355</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayout">
     <item row="7" column="0" colspan="3">
      <widget class="Line" name="line">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
     </item>
     <item row="11" column="1">
      <widget class="QLineEdit" name="lineEdit_host_password">
       <property name="focusPolicy">
        <enum>Qt::TabFocus</enum>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QLineEdit" name="lineEdit_src">
       <property name="focusPolicy">
        <enum>Qt::TabFocus</enum>
       </property>
       <property name="text">
        <string>System Director IP/Virtual IP</string>
       </property>
      </widget>
     </item>
     <item row="19" column="0">
      <spacer name="verticalSpacer">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>40</height>
        </size>
       </property>
      </spacer>
     </item>
     <item row="0" column="0">
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Source System Director:</string>
       </property>
      </widget>
     </item>
     <item row="0" column="2">
      <widget class="QPushButton" name="btn_migrate">
       <property name="text">
        <string>Migrate Filesystem</string>
       </property>
      </widget>
     </item>
     <item row="8" column="1">
      <widget class="QLineEdit" name="lineEdit_host_name"/>
     </item>
     <item row="2" column="2">
      <widget class="QPushButton" name="btn_recovery">
       <property name="text">
        <string>System Recovery</string>
       </property>
      </widget>
     </item>
     <item row="11" column="0">
      <widget class="QLabel" name="label_password">
       <property name="text">
        <string>Host Password</string>
       </property>
      </widget>
     </item>
     <item row="8" column="2">
      <widget class="QPushButton" name="btn_src_test">
       <property name="text">
        <string>Source connection Test</string>
       </property>
      </widget>
     </item>
     <item row="16" column="0">
      <widget class="QLabel" name="label_4">
       <property name="text">
        <string>Source activities</string>
       </property>
      </widget>
     </item>
     <item row="17" column="0">
      <widget class="QLabel" name="label_5">
       <property name="text">
        <string>Destination activities</string>
       </property>
      </widget>
     </item>
     <item row="5" column="1">
      <widget class="QLineEdit" name="lineEdit_mcpassword"/>
     </item>
     <item row="2" column="1">
      <widget class="QLineEdit" name="lineEdit_dest"/>
     </item>
     <item row="5" column="0">
      <widget class="QLabel" name="label_password_2">
       <property name="text">
        <string>Management Console Password:</string>
       </property>
      </widget>
     </item>
     <item row="11" column="2">
      <widget class="QPushButton" name="btn_dest_test">
       <property name="text">
        <string>Destination connection Test</string>
       </property>
      </widget>
     </item>
     <item row="5" column="2">
      <widget class="QPushButton" name="btn_cancel">
       <property name="text">
        <string>Cancel</string>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>Destination System Director:</string>
       </property>
      </widget>
     </item>
     <item row="8" column="0">
      <widget class="QLabel" name="label_username">
       <property name="text">
        <string>Host user name</string>
       </property>
      </widget>
     </item>
     <item row="21" column="0" colspan="3">
      <widget class="QSlider" name="horizontalSlider">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
     </item>
     <item row="20" column="0" colspan="3">
      <widget class="QTextEdit" name="textlog"/>
     </item>
     <item row="16" column="1" colspan="2">
      <widget class="QProgressBar" name="progressBar_src">
       <property name="value">
        <number>0</number>
       </property>
      </widget>
     </item>
     <item row="17" column="1" colspan="2">
      <widget class="QProgressBar" name="progressBar_dest">
       <property name="value">
        <number>0</number>
       </property>
      </widget>
     </item>
     <item row="1" column="2">
      <widget class="QPushButton" name="btn_name">
       <property name="font">
        <font>
         <pointsize>10</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Name</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar">
   <property name="autoFillBackground">
    <bool>true</bool>
   </property>
   <property name="inputMethodHints">
    <set>Qt::ImhNone</set>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>btn_migrate</sender>
   <signal>clicked()</signal>
   <receiver>statusbar</receiver>
   <slot>clearMessage()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>489</x>
     <y>22</y>
    </hint>
    <hint type="destinationlabel">
     <x>282</x>
     <y>273</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>btn_recovery</sender>
   <signal>clicked()</signal>
   <receiver>textlog</receiver>
   <slot>cut()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>489</x>
     <y>51</y>
    </hint>
    <hint type="destinationlabel">
     <x>285</x>
     <y>281</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
